src/index.tsx

```diff
@@ -16,6 +16,20 @@ function validateNode(node: ElementContext): Promise<Result[]> {
   });
 }
 
+// Copied from:
+// https://stackoverflow.com/questions/29321742/react-getting-a-component-from-a-dom-element-for-debugging
+// ¯\_(ツ)_/¯
+function getComponentFromNode(node: HTMLElement): string {
+  const [component] = Object.keys(node)
+    .filter(key => key.startsWith('__reactInternalInstance$'))
+    .map((key: string) => {
+      const fiberNode = (node as any)[key];
```

Comment:
I can't figure the types out here so I just told TS to shut up, for now.

Reply:
Don't be afraid to tell TS to shut it's annoying whining from time to time!

Reply:
How else would you expect it to learn some manners 😄